README.md

```diff
@@ -65,16 +55,17 @@ Run the pre-built `runtime` container:
 DOCKER_IMAGE_TAG=22.06-runtime ./docker/run_container_release.sh
```

Comment:
This is where people will stub their toes.

"If you are not building the runtime container locally from the current git branch, use an applicable tag from NGC. For example, `DOCKER_IMAGE_TAG=22.04-runtime`.
More official release tags can be found at https://catalog.ngc.nvidia.com/orgs/nvidia/teams/morpheus/containers/morpheus/tags"

Reply:
But do we want a ref to 22.04 in the 22.06 docs?